src/WebApi/DependencyInjection.cs

```diff
@@ -13,9 +13,9 @@ public static void AddWebApi(this IServiceCollection services, IConfiguration co
     {
         services.AddHttpContextAccessor();
         services.AddScoped<ICurrentUserService, CurrentUserService>();
-
-        services.AddOpenApiDocument(configure => configure.Title = "CleanArchitecture API");
+        
         services.AddEndpointsApiExplorer();
```

Comment:
Can you check if we definitely need the call to `AddEndpointsApiExplorer()` ?

Reply:
.AddOpenApi() adds it internally, so not necessary anymore. Will remove.

![image](https://github.com/user-attachments/assets/8ddf54d4-7e30-475a-a760-72b1eee8c7cc)

Reply:
Done.